src/toM3u8.js

```diff
@@ -40,25 +43,24 @@ const mergeDiscontiguousPlaylists = playlists => {
   });
 };
 
-const addSegmentInfoFromSidx = (playlists, sidxMapping = {}) => {
+export const addSidxSegmentsToPlaylist = (playlist, sidxMapping) => {
```

Comment:
Add Sidx segments to a single playlist. We used to do this only on an array of playlists, but it was weird because for audio/vtt playlists we needed to pass in a single playlist. The naming of these functions was also weird.